static/templates/compose_banner.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="compose_banner {{banner_type}}" id="{{css_id}}" data-stream-id="{{stream_id}}" data-topic-name="{{topic_name}}">
```

Comment:
Maybe `element_id` would be better than `css_id`, since it's technically an HTML element ID, not a CSS thing.